lerna.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Is lerna still needed?

Reply:
I haven't dealt with NPM workspaces in the context of a published module yet. My guess is no, lerna is no longer needed. I kept it around in case publishing becomes complicated for some reason.

Reply:
All the npm commands now have a `--workspaces` flag, so you can use that to update versions (or do it manually), and use it to publish!

I’d say remove lerna?

Reply:
Removed lerna